README.md

```diff
@@ -61,7 +79,7 @@ This limit is imposed upon by the Bluetooth adapter hardware as well as it's fir
 
 | Platform |     |
 | :------- | --- |
-| OS X 10.11 (El Capitan) | 6 |
+| macOS 10.11 (El Capitan) | 6 |
 | Linux/Windows - Adapter dependent | 5 (CSR based adapter) |
```

Comment:
Can you please leave "OS X" in for the read me for now? I'm not sure what the new limit is on macOS Sierra.